rclrs_example_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Instead of adding `rclrs_example_msgs`, I'd prefer using the `example_interfaces` or `test_interface_files` packages.

Reply:
We need https://github.com/ros2/test_interface_files/pull/18 to make that work, I believe

Reply:
I'd propose to merge the version with the `rclrs_example_msgs` if the PR I linked isn't approved by the time the rest of the review is finished, and then later remove it once it is.

Reply:
Seems like bringing in `test_interface_files` is more complicated than we thought. Would you be fine with leaving in `rclrs_example_msgs` for now?